models/magiceden/solana/magiceden_solana_events.sql

```diff
@@ -84,18 +84,22 @@ SELECT
   +abs(post_balances[14] / 1e9 - pre_balances[14] / 1e9)
   +abs(post_balances[15] / 1e9 - pre_balances[15] / 1e9)) / ((abs(post_balances[0] / 1e9 - pre_balances[0] / 1e9)-0.00204928)) * 100),2) as royalty_fee_percentage,
   NULL::double as royalty_fee_receive_address,
-  CASE WHEN (array_contains(account_keys, 'M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K')) 
-         AND array_contains(log_messages, 'Program log: Instruction: ExecuteSale') 
-         AND array_contains(log_messages, 'Program log: Instruction: Buy') THEN 'SOL' 
+  CASE WHEN (array_contains(account_keys, 'M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K'))
+         AND array_contains(log_messages, 'Program log: Instruction: ExecuteSale')
+         AND array_contains(log_messages, 'Program log: Instruction: Buy') THEN 'SOL'
          ELSE NULL::string END as royalty_fee_currency_symbol,
   signatures[0] || '-' || id || '-' || instructions[0]::string as unique_trade_id,
-  instructions, 
+  instructions,
   signatures,
   log_messages
 FROM {{ source('solana','transactions') }}
-LEFT JOIN prices.usd p 
+LEFT JOIN prices.usd p
   ON p.minute = date_trunc('minute', block_time)
-  AND p.symbol = 'SOL'        
+  AND p.blockchain is NULL
+  AND p.symbol = 'SOL'
+  {% if is_incremental() %}
+  AND p.minute >= date_trunc("day", now() - interval '1 week')
```

Comment:
Same as previous comment here (not going to comment the same thing everywhere but same thought process)